packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -105,6 +129,95 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
         return result;
     }
 
+    private static validateKeyUniquenessOfRelationshipAttributesWithinRequest(items: (RequestItem | RequestItemGroup)[], recipient?: CoreAddress) {
+        const fragmentsOfMustBeAcceptedItemsOfRequest: RelationshipAttributeFragment[] = [];
+        for (const item of items) {
+            if (item instanceof RequestItemGroup) {
+                const fragmentsOfMustBeAcceptedItemsOfGroup = OutgoingRequestsController.extractRelationshipAttributeFragmentsFromMustBeAcceptedItemsOfGroup(item, recipient);
```

Comment:
What is the exact use for restricting this to the items that must be accepted? I think that items that can't be accepted shouldn't be created. Is it about creating e. g. two ReadAttributeRequestItems for the same key with different queries where only one can be accepted?

Reply:
You are right, I have restricted it to `mustBeAccepted` RequestItems due to some edge cases. :D

For example, the RequestItem could become acceptable again if the existing RelationshipAttribute of this `key` has been deleted in the meantime. Even if it has not been deleted, the Request is not immediately unusable because the RequestItem can still be rejected as the RequestItem is not `mustBeAccepted`.

I think it's debatable whether this makes sense or not. In general, I have tried to validate as little restrictively as possible.

Reply:
I understand, also agree that it's debatable. I don't know what kind of requests are commonly sent, so I don't really have an opinion there